auth-web/package-lock.json

```diff
@@ -8131,6 +8131,7 @@
       "version": "1.4.0",
       "inBundle": true,
       "license": "MIT",
+      "optional": true,
```

Comment:
Hmm, I'm using node 16.15.0

Reply:
I think https://github.com/bcgov/sbc-auth/pull/2569 changed it previously.